alma-integrations/src/main/java/no/sikt/rsp/ContactInfoConverter.java

```diff
@@ -30,6 +35,23 @@ public final class ContactInfoConverter {
                                                               "shipping");
     private static final List<String> EMAIL_TYPES = List.of("claimMail", "orderMail", "paymentMail", "queries",
                                                             "returnsMail");
+    public static final String LOCALES_WITHOUT_ISO_3_COUNTRY_CODES = "Locales without ISO3Country-codes: ";
+    public static final String COMMA_SEPARATOR = ", ";
+    static final Map<String, String> twoToThreeMap = new HashMap<>();
+
+    static {
```

Comment:
Var ikke dette allerede gjort i LUM? Har vi samme kode to plasser nå?

Reply:
Fikset